cluster-autoscaler/cloudprovider/gce/gce_price_info.go

```diff
@@ -513,6 +518,8 @@ var (
 		"t2d-standard-32":  0.3271,
 		"t2d-standard-48":  0.4907,
 		"t2d-standard-60":  0.6134,
+		"z3-highmem-88":    6.75,
+		"z3-highmem-176":   10.37,
```

Comment:
prices we use are for us-cental1.
Could you change that this to reflect that.

https://cloud.google.com/compute/vm-instance-pricing

Reply:
Done.